Updates Management doc to reflect new navigation (#67506)
elastic/kibana: docs/user/management.asciidoc

indices, clusters, licenses, UI settings, index patterns, spaces, and more.

[float]
[[manage-ingest]]
== Ingest

[cols="50, 50"]
|===

| <<ingest-node-pipelines, Ingest Node Pipelines>>
| Create and manage {es}
pipelines that enable you to perform common transformations and
enrichments on your data.

| {logstash-ref}/logstash-centralized-pipeline-management.html[Logstash Pipelines]
| Create, edit, and delete your Logstash pipeline configurations.

> | <<managing-beats, Beats Central Management>>

Reviewer: This feature has been discontinued, so it might not belong here.

Reviewer: Nice work on this section!

years of historical data in combination with your raw data.

| {ref}/transforms.html[Transforms]
|Use transforms to pivot existing {es} indices into summarized or entity-centric indices.

| <<managing-cross-cluster-replication, Cross-Cluster Replication>>
|Replicate indices on a remote cluster and copy them to a follower index on a local cluster.
This is important for
disaster recovery. It also keeps data local for faster queries.

| <<working-remote-clusters, Remote Clusters>>
|Manage your remote clusters for use with cross-cluster search and cross-cluster replication.
You can add and remove remote clusters, and check their connectivity.
|===

[float]
[[manage-alerts-insights]]
== Alerts and Insights

[cols="50, 50"]
|===

| <<managing-alerts-and-actions, Alerts&nbsp;and Actions>>
| Centrally manage your alerts across {kib}. Create and manage reusable
connectors for triggering actions.

| <<reporting-getting-started, Reporting>>
| Monitor the generation of reports&mdash;PDF, PNG, and CSV&mdash;and download reports that you previously generated.
A report can contain a dashboard, visualization, saved search, or Canvas workpad.

| <<xpack-ml-anomalies, Machine Learning Jobs>>

> | Create jobs that model the normal behavior of your time series data &mdash;

Reviewer 2: It looks like you can only view the list of jobs and launch the ML UIs from here, so I suggest something like this:
```suggestion
| View your {anomaly-jobs} and {dfanalytics-jobs}. Open the Single Metric Viewer or Anomaly Explorer to see your {ml} results.
```

learning trends, periodicity, and more &mdash; so you can detect anomalies,

> streamline root cause analysis, and reduce false positives.

Reviewer 2: > Machine Learning Jobs

I just double-checked and this link takes you to both anomaly detection jobs and data frame analytics jobs, so I think the description needs to be more appropriate for both types.

| <<watcher-ui, Watcher>>
| Detect changes in your data by creating, managing, and monitoring alerts.
For example, create an alert when the maximum total CPU usage on a machine goes
above a certain percentage.

|===

[float]
[[manage-security]]
== Security

[cols="50, 50"]
|===

a| <<xpack-security, Users>>
|View the users that have been defined on your cluster.
Add or delete users and assign roles that give users
specific privileges.

| <<xpack-kibana-role-management, Roles>>
|View the roles that exist on your cluster. Customize

> the actions that a user with the role can perform, both on a cluster and an index level.

Reviewer 3: This also allows you to grant access to Kibana

```suggestion
the actions that a user with the role can perform, on a cluster, index, and space level.
```
